sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ChangeFeedProcessorBuilder.java

```diff
@@ -133,6 +133,33 @@ public ChangeFeedProcessorBuilder handleChanges(Consumer<List<JsonNode>> consume
         return this;
     }
 
+    /**
+     * Sets a consumer function which will be called to process changes for LatestVersion change feed mode.
+     *
+     * <!-- src_embed com.azure.cosmos.latestVersionChanges.handleChanges -->
+     * <pre>
+     * .handleLatestVersionChanges&#40;docs -&gt; &#123;
+     *     for &#40;JsonNode item : docs&#41; &#123;
+     *         &#47;&#47; Implementation for handling and processing of each JsonNode item goes here
+     *     &#125;
+     * &#125;&#41;
+     * </pre>
+     * <!-- end com.azure.cosmos.latestVersionChanges.handleChanges -->
+     *
+     * @param consumer the {@link Consumer} to call for handling the feeds.
+     * @return current Builder.
+     */
+    @Beta(value = Beta.SinceVersion.V4_40_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public ChangeFeedProcessorBuilder handleLatestVersionChanges(Consumer<List<JsonNode>> consumer) {
```

Comment:
should we return `JsonNode` or `ChangeFeedProcessorItem`

Reply:
ChangeFeedProcessorItem (to normalizer eventually on public surface area) - but to be honest I don't particularly like the name of that class (probably too late) - ChangeFeedNotification / event/ record would probably describe better what it is. Has the current beta-api naming been discussed with central team already? Or will we have the naming discussion anyway with them? If the former definitely ignore my feedback (not worth changing now). Even if we have not discussed it yet I would only consider it if there are others with similar feedback.

Reply:
I think it would be good to add a comment that this method is a replacement for the old handleChanges (that will eventually be deprecated) - but that the semantics are compatible